sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/ObserverExceptionWrappingChangeFeedObserverDecorator.java

```diff
@@ -7,13 +7,18 @@
 import com.azure.data.cosmos.internal.changefeed.ChangeFeedObserverCloseReason;
 import com.azure.data.cosmos.internal.changefeed.ChangeFeedObserverContext;
 import com.azure.data.cosmos.internal.changefeed.exceptions.ObserverException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Mono;
 
 import java.util.List;
 
 /**
  * Exception wrapping decorator implementation for {@link ChangeFeedObserver}.
  */
 class ObserverExceptionWrappingChangeFeedObserverDecorator implements ChangeFeedObserver {
+    private final Logger logger = LoggerFactory.getLogger(ObserverExceptionWrappingChangeFeedObserverDecorator.class);
```

Comment:
static final logger ?

Reply:
In the other SDK code it looks like we both "private final" and "private static final". Let me know which one we should use one or the other in consistent manner.

Reply:
as a rule of thumb if you regularly create many instances of this class having a static logger may help to reduce CPU overhead in CPU intensive path. If the instance of this class is long-lived like in the Client then no need to be static.

https://www.slf4j.org/faq.html#declared_static

Reply:
There are couple classes that are expected to have multiple instances. I've made the logger "static" for those cases.